packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -736,15 +839,25 @@ const Conversation = WebexPlugin.extend({
     }, activity);
   },
 
+  /**
+   * Update an existing activity
+   * @param {Object} conversation
+   * @param {Object} object
+   * @param {Object} activity
+   * @returns {Promise}
+   */
   update(conversation, object, activity) {
```

Comment:
We sure love our ambiguous `object` in the SDK. I know it isn't your naming convention, but probably could use some more help in the jsdoc at least